src/main/java/reposense/git/GitDownloader.java

```diff
@@ -15,12 +15,12 @@
 
     private static final Logger logger = LogsManager.getLogger(GitDownloader.class);
 
-    public static void downloadRepo(String organization, String repoName, String branchName)
+    public static void downloadRepo(String location, String displayName, String repoName, String branchName)
```

Comment:
a bit strange that downloadRepo isn't using RepoConfig directly, alot of the code within it can be simplified.
E.g. `GitChecker.checkout(config.getRepoRoot())`